Makefile

```diff
@@ -117,7 +117,7 @@ verify_goimports:
 benchmark:
 	go test -mod vendor -run=XXX -bench=. ./...
 
-errcheck:
+golangci-lint:
```

Comment:
is there much downside to folding this into the lint target? having both `lint` and `golangci-lint` feels weird, and I always get annoyed trying to remember what wordbreaking they used (e.g., go-lang-ci-lint or golang-ci-lint or golang-cilint...four words, one hyphen!)

Reply:
yeah I plan on folding it in soon. We can actually replace all of the existing lint stuff with `golangci-lint`.